services/app-api/utils/constants/constants.ts

```diff
@@ -18,6 +18,7 @@ export const error = {
   NOT_IN_DATABASE: "Record not found in database.",
   MISSING_FORM_TEMPLATE: "Form Template not found in S3.",
   MISSING_FIELD_DATA: "Field Data not found in S3.",
+  NO_WORKPLANS_FOUND: "No record of work plans found in database",
```

Comment:
```suggestion
  NO_WORKPLANS_FOUND: "No record of Work Plans found in database",
```
nit: just some consistency on the Work Plans title! no biggie.